server/deploy/helm/charts/lomas_server/templates/server/service.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   type: {{ .Values.server.service.type }}
   ports:
   - port: {{ .Values.server.service.port}}
-    targetPort: http
```

Comment:
i wonder why this works ?

Reply:
ah ok, the names 'http' match and then everything is 80, maybe it is better to differentiate the service port as 80 and container port as 8080

Reply:
Yes, we could allow to specify the target port in the values file too.